### ISSUE TITLE
Player can spectate other players by clicking on them when unconscious

### ISSUE THREAD

**Issue by Avokadomos** [ArmaSweden/Arma-Sweden-Extras#5]
Clicking on other players in spectator mode changes view to them. Clicking should be disabled, or spectator should be reconfigured to prevent this.

**Comment by Avokadomos** [comment]
Making whitelisted sides empty results in player being able to spectate everyone. There is no parameter to remove all sides. This requires forcing the whitelisted variable to something other than faction sides.

```sqf
missionNamespace setVariable ["BIS_EGSpectator_whitelistedSides", [objNull]]
```

This will result in spectator being unable to click other units in 3D space.

**Comment by Avokadomos** [comment]
Fixed in a640c18cc4cdb8c593100174f52d13c4f7e3a7ea.
